docs/guides/use_checkpointing.md

```diff
@@ -1,67 +1,85 @@
 ---
-jupytext:
-  formats: ipynb,md:myst
-  text_representation:
-    extension: .md
-    format_name: myst
-    format_version: 0.13
-    jupytext_version: 1.13.8
-kernelspec:
-  display_name: Python 3 (ipykernel)
-  language: python
-  name: python3
+jupyter:
+  jupytext:
+    formats: ipynb,md
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.13.8
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
 ---
 
-# Checkpointing with `flax.training.checkpoints`
+# Save / Load Checkpoints
 
 In this guide, you will learn about [`flax.training.checkpoints`](https://flax.readthedocs.io/en/latest/api_reference/flax.training.html#module-flax.training.checkpoints)—a simplistic and generic checkpointing library built into Flax. With Flax Checkpoints, you can save and load model parameters, metadata, and a variety of Python data. In addition, it provides basic features for versioning, automatic bookkeeping of past checkpoints, and asynchronous saving to reduce training wait time.
 
+> **_Note:_** In the foreseeable future, Flax's checkpointing functionality will gradually be migrated to [Orbax](https://github.com/google/orbax), which provides a more customizable and flexible API for various array types and storage formats. All existed features of Flax will continue to be supported, though the API may be slightly changed. This page will contain the most up-to-date code examples to use Orbax-Flax checkpointing.
```

Comment:
Suggestions:

- "All existing features in the Flax API will continue to be supported, though the API may change."
- "Once the migration is complete, this guide will contain the most up-to-date code examples with Orbax-Flax checkpointing."

```suggestion
> **_Note:_** In the foreseeable future, Flax's checkpointing functionality will gradually be migrated to [Orbax](https://github.com/google/orbax), which provides a more customizable and flexible API for various array types and storage formats. All existing features in the Flax API will continue to be supported, though the API may change. This page will contain the most up-to-date code examples to use Orbax-Flax checkpointing.
```